### PR TITLE
Review of the APS state

### PR DESCRIPTION
**Pull request by italobusi**
We have just noted that if we remove the protection-external-commands container, there is no more information about whether a lockout-of-protection, forced-switch and manual-switch APS state is driven by a local or a remote operator's command and therefore can be cleared by local clear operator command

We have also noted that we have forgotten the manual-switch APS state

Italo and Sergio

### CODE REVIEW

File: ietf-te-types.yang [hunk 1]
```diff
@@ -503,18 +503,34 @@ module ietf-te-types {
...
     base lsp-protection-state;
     description
-        "A Loss of Protection (LoP) command is active.";
+        "A local Loss of Protection (LoP) command is active.";
     reference
         "ITU-T G.808, RFC 4427";
   }
 
-  identity forced-switch {
+  identity remote-lockout-of-protection {
```
Comment (ietf-mpls-yang) [inline ietf-te-types.yang]: local and remote is not clear -
maybe ingress/egress is better

File: ietf-te-types.yang [hunk 2]
```diff
@@ -503,18 +503,34 @@ module ietf-te-types {
...
     base lsp-protection-state;
     description
-        "A forced switch (FS) command is active.";
+        "A remote Loss of Protection (LoP) command is active.";
+    reference
+        "ITU-T G.808, RFC 4427";
+  }
+	
+  identity local-forced-switch {
```
Comment (ietf-mpls-yang) [inline ietf-te-types.yang]: suggest splitting into 2:
- action
- at which node the action was applied

Comment (ietf-mpls-yang) [reply ietf-te-types.yang]: node may be identified by node-id?
The clear command can be only be allowed if applied on the same node-id

Comment (sergiobelotti) [reply ietf-te-types.yang]: The scenario we need to understand is the one in which the command e.g. lockout-of-protection is applied to a node that is outside the control of the server e.g. a node controller or a PNC. That means that this server cannot remove the operator command when it receives a   "clear" action from its client. So the problem is not related to control the node to which the server should apply an operator command but to report whether the operator command has been applied to a node controlled by the server (e.g. PNC)  or not.

**Comment by sergiobelotti** [comment]
Attached a slide describing the process and the issue.
[tunnel-protection-local-remote-state.pptx](https://github.com/ietf-mpls-yang/te/files/1576101/tunnel-protection-local-remote-state.pptx)

To summarize the scenario: 

- Node 1 if receives "Clear" command will delete local LoP 

- PNC A if receives "Clear command" , will forward command to Node 1 

- MDSC if receives from client/application/CNC a "Clear" command , will forward it to PNC A

- PNC B and node 6 , if they receive a "Clear" command, they do nothing , the state of LoP remains unchanged.

 

-

File: ietf-te-types.yang (hunk 2 above)
Comment (ietf-mpls-yang) [reply ietf-te-types.yang]: action
node protection-group-ingress or/both
node protection-group-egress

and in state, 2 leafs:
node protection-group-ingress or/both
node protection-group-egress

**Pull request closed without merging**